tools/extend_sr.php

```diff
@@ -0,0 +1,58 @@
+<?php
+$relPath="../pinc/";
+include_once($relPath.'base.inc');
+include_once($relPath.'theme.inc');
+include_once($relPath.'Project.inc');
+include_once($relPath.'project_states.inc');
+include_once($relPath.'smooth_read_change.inc'); // handle_smooth_reading_change()
+
+class ActionException extends Exception {}
+
+require_login();
+
+$projectid = validate_projectID('project', @$_REQUEST['project']);
+$project = new Project($projectid);
+$days = get_integer_param($_REQUEST, 'days', 0, 0, 56);
+
+$title = _("Extend the Smooth Reading deadline");
+
+output_header($title);
```

Comment:
You might consider using `slim_header()` here with: if an error occurs do what you do now. If success use `metarefresh()` with 1 second to route the user back to the project page with just the success message showing briefly (it passes in as an argument to `metarefresh()`). That's similar to how other interstitial pages work and I think would be a little more consistent.

Reply:
Done this. It is consistent with the way upload_text.php works, but other upload cases such as handle_bad_page and remote file manager show a persistent confirmation.

Reply:
Thanks. I've created https://www.pgdp.net/c/tasks.php?action=show&task_id=1797 to track standardizing how we do this, because you're right - it's all across the board.